platform/ui/src/components/NavBar/NavBar.mdx

```diff
@@ -0,0 +1,32 @@
+---
+name: NavBar
+menu: Components
+route: components/navBar
+---
+
+import { Playground, Props } from 'docz';
+import Button from './../Button';
```

Comment:
Let's import these components from `@ohif/ui` instead of its relative path.

`Button` is not being used? Remove if not.

Reply:
True. Did this way because we were doing this way. hehe 

Just updated all docz pages that were not importing ohif/ui components not from @ohif/ui

Reply:
\o/ \o/ \o/ Thanks!